pkg/doc/signature/verifier/verifier.go

```diff
@@ -48,14 +48,14 @@ type keyResolver interface {
 
 // DocumentVerifier implements JSON LD document proof verification
 type DocumentVerifier struct {
-	signatureSuites []signatureSuite
+	signatureSuites []SignatureSuite
 	pkResolver      keyResolver
 }
 
 // New returns new instance of document verifier
-func New(resolver keyResolver, mainSuite signatureSuite, extraSuites ...signatureSuite) *DocumentVerifier {
+func New(resolver keyResolver, mainSuite SignatureSuite, extraSuites ...SignatureSuite) *DocumentVerifier {
```

Comment:
Why do we have distinction between mainSuite and extraSuites? And how should the consumer decide between main and extra ?

Reply:
I used this approach to force a definition at least one signature suite. There is no difference between the main and extra signature suites.

Reply:
I agree that we shouldn't have mainSuite and extra suites. You can provide a comment saying that at least one suite is required and check for suites length in New().

Reply:
the terminology "main" and "extra" is likely to be confusing.

Reply:
@rolsonquadras @sandrask @troyronda 
Created https://github.com/hyperledger/aries-framework-go/issues/1616